aldryn_search/search_indexes.py

```diff
@@ -26,7 +26,12 @@ def prepare_login_required(self, obj):
         return obj.page.login_required
 
     def prepare_site_id(self, obj):
```

Comment:
we should probably do a condition to select related the page node if we're on 3.5, otherwise this would result in quite a lot of queries.